2.6.md

```diff
@@ -850,6 +850,16 @@ This object contains any legal, governmental, or industry regulations that the s
     <td>string</td>
     <td>Communicates signals regarding consumer privacy under US privacy regulation. See <a href="https://github.com/InteractiveAdvertisingBureau/USPrivacy/blob/master/CCPA/US%20Privacy%20String.md">US Privacy String specifications</a>. Refer to Section 7.5 for more information.</td>
   </tr>
+  <tr>
+    <td><code>gpp</code></td>
+    <td>string</td>
+    <td>Contains the Global Privacy Platform’s consent string. See the <a href="https://github.com/InteractiveAdvertisingBureau/Global-Privacy-Platform">Global Privacy Platform specification</a> for more details.</td>
+  </tr>
+  <tr>
+    <td><code>gpp_sid</code></td>
+    <td>integer array</td>
+    <td>Array of the section(s) of the string which should be applied for this transaction. Generally will contain one and only one value, but there are edge cases where more than one may apply. GPP Section 3 (Header) and 4 (Signal Integrity) do not need to be included. See the <a href="https://github.com/InteractiveAdvertisingBureau/Global-Privacy-Platform/blob/main/Sections/Section%20Information.md">GPP Section Information</a> for more details.</td>
```

Comment:
For completeness, does this mean the caller _may_ include 3 or 4 or both in this list, and the receiver should therefore ignore these values when they appear?

If sections 3 and 4 are not to be mentioned (because they are required), we can strengthen the language here from "do not need" to "are not".

Reply:
I agree with Simon here, it's unclear what to do with section 3 if we have it from this language